### ISSUE TITLE
Previous tab in settings.json doesnt work

### ISSUE THREAD

**Issue by yawhide** [microsoft/terminal#11091]
### Windows Terminal version (or Windows build number)

1.9.1942.0 & 1.10.2383.0

### Steps to reproduce

Edit the settings.json file and add

```
{
            "command": "previousTab",
            "keys": "alt+shift+["
        }
```
to your actions array.

Previous tab will not be updated correctly.
![image](https://user-images.githubusercontent.com/5014112/131548960-4286bef8-908e-4c17-b096-7ec1cd34d600.png)

### Expected Behavior

Previous tab hotkey would be remapping to `alt+shift+[`

### Actual Behavior

Previous tab is hotkey is not changing to what I specified in my settings.json file

**Comment by DHowett** [comment]
If you actually press <kbd>alt+shift+[</kbd>, does it work?

**Comment by yawhide** [comment]
It doesnt. It just types those characters or does weird stuff like w.e alt+shift+[ does

**Comment by DHowett** [comment]
_Huh_. This might be a documentation miss.

It looks like this is what I got out of the settings editor:

![image](https://user-images.githubusercontent.com/189190/131556012-fb62de60-bf03-4a1b-bd40-60c2de7d6810.png)

(prevTab instead of previousTab)

Does switching to `prevTab` work for you? You don't need to do the command/action nonsense, you can just use `"command": "prevTab"`

**Comment by yawhide** [comment]
> _Huh_. This might be a documentation miss.
> 
> It looks like this is what I got out of the settings editor:
> 
> ![image](https://user-images.githubusercontent.com/189190/131556012-fb62de60-bf03-4a1b-bd40-60c2de7d6810.png)
> 
> (prevTab instead of previousTab)
> 
> Does switching to `prevTab` work for you? You don't need to do the command/action nonsense, you can just use `"command": "prevTab"`

where did you find that previous tab was actually prevTab? Is there online documentation on this?

also prevTab works, thank you :)

**Comment by DHowett** [comment]
I'm glad it worked! This is mentioned in a couple places:

1. the settings schema document, so VS Code will offer it as an example:
2. The documentation, in the Actions section, heading [Open Previous tab](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-previous-tab)
3. The Settings UI, sort-of. That's how I actually got the entry I showed you. Version 1.10 (preview) and 1.11 (preview) let you do this:

![image](https://user-images.githubusercontent.com/189190/131558358-c98b6efd-e775-4401-bd90-b58329942f7f.png)

Unfortunately, that feature isn't available in Stable yet.
